internal/rbd/driver.go

```diff
@@ -194,5 +194,13 @@ func (r *Driver) Run(conf *util.Config) {
 		util.DebugLogMsg("Registering profiling handler")
 		go util.EnableProfiling()
 	}
+	if conf.IsNodeServer {
+		go func() {
+			err := RunVolumeHealer(r.ns, conf)
+			if err != nil {
+				util.ErrorLogMsg("healer had failures, err %v\n", err)
```

Comment:
I hope the healing will be moved to a **csi-addons sidecar** that will call `NodeStageVolume` like other standard CSI procedures. It might be worth to report that as an issue at https://github.com/csi-addons/spec so we can address it in the future. Just like other issues in the csi-addons spec, there could be a CRD/job that recovers (or force removes) a VolumeAttachment.

Reply:
Done @nixpanic, here is the tracker https://github.com/csi-addons/spec/issues/20